cmds/ip/ip.go

```diff
@@ -49,16 +49,31 @@ func usage() {
 		arg[0:cursor], arg[cursor:], arg[cursor], whatIWant)
 }
 
+func one(cmd string, cmds []string) string {
+	var x, n int
+	for i := range cmds {
+		if strings.HasPrefix(cmds[i], cmd) {
+			n++
+			x = i
+		}
+	}
+	if n == 1 {
+		return cmds[x]
+	}
+	return "NO MATCH"
```

Comment:
Use a bool instead of this?

Reply:
Or the empty string, if you want to keep the switch stmts simple.

Reply:
I went with "". Thanks!